kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md

```diff
@@ -61,9 +61,9 @@ The PV and PVC creation inputs can be customized by editing the `create-pv-pvc-i
 
 ## Shared versus dedicated PVC
 
-By default, the `domainUID` is left empty in the inputs file so that the generated PV and PVC can be shared by multiple domain resources in the same Kubernetes namespaces.
+By default, the `domainUID` is left empty in the inputs file, which means the generated PV and PVC will not be tied to any particular domain. In other words, the generated PV and PVC can be shared by multiple domain resources in the same Kubernetes namespaces as the PV and PVC.
```

Comment:
will not be tied to any particular domain -> will not be associated with a particular domain, but can be shared by multiple domain resources in the same Kubernetes namespace as the PV and PVC.